client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-upgrade-plan/index.tsx

```diff
@@ -28,28 +29,41 @@ const SiteMigrationUpgradePlan: Step = function ( { navigation, data } ) {
 		return;
 	}
 
+	const queryParams = new URLSearchParams( window.location.search );
```

Comment:
In the other flow we also use `props?.data?.path`, should we check data?.path here as well?

Reply:
In this context, there is no `data.path` or any prop we could use. I think it is fine to rely on location.search as well.

Reply:
Added `useQuery` here too 👍